### PR TITLE
Add tests for Editable.adaptFormatter()

### PR DESCRIPTION
**Pull request by BE-Webdesign**
Add tests for Editable.adaptFormatter()

### CODE REVIEW

**Comment by gziolo** [comment]
I have one concern here. This test covers private function from the component. There was usually a rule to test only public API in the previous projects, I worked with. Do you think we should keep the same rule in Gutenberg? We can always move that method to the file with utilities when we fill that it deserves its own tests.

cc @aduth @youknowriad @mtias

**Comment by gziolo** [comment]
If we are fine with testing private methods, then it's good to go 👍

**Comment by BE-Webdesign** [comment]
One option is if these private methods are used somewhere in a public facing code we can get coverage that way. For now going to leave the PR open.

File: blocks/editable/test/index.js [hunk 1]
```diff
@@ -10,6 +10,24 @@ import Editable from '../';
...
+				type: 'inline-style',
+				style: {
+					'font-weight': 'bold',
+				},
+			};
+
+			test( 'should return an object on inline: span, and a styles property matching the style object provided', () => {
+				expect( wrapper.instance().adaptFormatter( options ) ).toEqual( {
+					inline: 'span',
+					styles: { ...options.style },
```
Comment (gziolo) [inline blocks/editable/test/index.js]: It can also simply `styles: options.style,`. No need to clone it.

**Review by gziolo — APPROVED** [review]
Let's merge it, we can change our approach about testing private methods later.

File: blocks/editable/test/index.js (hunk 1 above)
Comment (BE-Webdesign) [reply blocks/editable/test/index.js]: Sometimes I do silly things 🙃

**Pull request merged**